pkg/state/transaction_checker.go

```diff
@@ -1103,13 +1103,20 @@ func (tc *transactionChecker) checkCreateAliasWithProofs(transaction proto.Trans
 	if err := tc.checkFee(transaction, assets, info); err != nil {
 		return nil, err
 	}
-	activated, err := tc.stor.features.newestIsActivated(int16(settings.SmartAccounts))
+	smartAccountsIsActivated, err := tc.stor.features.newestIsActivated(int16(settings.SmartAccounts))
 	if err != nil {
 		return nil, err
 	}
-	if !activated {
+	if !smartAccountsIsActivated {
 		return nil, errors.New("SmartAccounts feature has not been activated yet")
 	}
+	rideV6IsActivated, err := tc.stor.features.newestIsActivated(int16(settings.RideV6))
+	if err != nil {
+		return nil, err
+	}
+	if tx.Proofs.MultiSigned() && !rideV6IsActivated {
```

Comment:
So we just decline not simple signed transactions before Ride V6 to match scala node?